### ISSUE TITLE
false positive: imagewebp

### ISSUE THREAD

**Issue by WDD-Marv** [vimeo/psalm#190]
with vimeo/psalm:dev-master, b4542316faf1a46801e8874ee8b02bfa20e4e53c

```php
imagewebp($im, $tmp);
```
> TooFewArguments - src\Transformer.php:65 - Too few arguments for method imagewebp

param 3 is *optional* integer for quality

related to phpstan/phpstan#389

**Comment by muglug** [comment]
Thanks, fixed!
